packages/beeq/src/components/page-title/bq-page-title.tsx

```diff
@@ -111,6 +111,7 @@ export class BqPageTitle {
         <div class={{ flex: true, '!hidden': !this.haveBackNavigation }} part="back">
           <slot name="back">
             <bq-button
+              class="hover:[&::part(button)]:bg-transparent"
```

Comment:
Why? 🤔

Reply:
Initially, I thought that we did not need hover effect on the arrow as I did not see it in the Figma design, but since it's a `button`, it make sense to have this hover, so I will delete it. 😎